package.json

```diff
@@ -93,7 +87,6 @@
     "bluebird": "^3.2.2",
     "browserify": "^12.0.2",
     "browserify-shim": "^3.8.12",
-    "browserify-versionify": "^1.0.6",
```

Comment:
at some point, we'll want to bring this back, at that time it should be a dependency and not a dev dep.